datahub-frontend/conf/application.conf

```diff
@@ -30,6 +30,12 @@ play.modules.enabled += "auth.AuthModule"
 play.allowHttpContext = true
 play.allowGlobalApplication = true
 
+# Uncomment to override server provider and set the max header count to a higher value
+# This is useful while using proxies like Envoy that result in the frontend server rejecting GMS
+# responses as there's more than the max of 64 allowed headers
+#play.server.provider = server.CustomAkkaHttpServerProvider
+#play.http.server.akka.max-header-count = 256
```

Comment:
Same kind of idea here, but this one for sure should have an env variable associated with it:

play.http.server.akka.max-header-count = ${DATAHUB_AKKA_MAX_HEADER_COUNT:64}

and if possible a reference to this in our configuration docs 😄

Reply:
Yeah agree, I can update this. Which docs would you suggest updating to surface this? I hunted for a couple of configs and I see them listed in some of the docker compose yaml files as well as the frontend.env file.

Reply:
Hmm, yeah I'm not seeing a current good spot for it, most of our configuration documentation seems to be for specific cases. I think just updating it in the frontend.env should be good enough for now and I'll create a ticket on our side to update the docs for this.